### PR TITLE
[DYN-3975] Fix non clickable textboxes

### PR DESCRIPTION
**Pull request by SHKnudsen**
### Purpose

this PR fixes: 
- https://jira.autodesk.com/browse/DYN-3976
- https://jira.autodesk.com/browse/DYN-3975

image
DynamoRefresh_groups_doubleClickBug

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers

@QilongTang 

### FYIs

@mjkkirschner

### CODE REVIEW

**Comment by QilongTang** [comment]
@SHKnudsen Please remake this PR against https://github.com/DynamoDS/Dynamo/tree/UIRefreshGroupImprovements

File: src/DynamoCoreWpf/Properties/Resources.Designer.cs [hunk 1]
```diff
@@ -2819,15 +2819,6 @@ public class Resources {
             }
         }
         
-        /// <summary>
-        ///   Looks up a localized string similar to Add group to group.
-        /// </summary>
-        public static string GroupContextMenuAddGroupToGroup {
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Properties/Resources.Designer.cs]: Is this change intended?

**Comment by SHKnudsen** [comment]
yea currently doing that
